lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -538,11 +565,24 @@ void SessionStateEnforcer::prepare_response_to_access(
     std::string upf_ip, uint32_t upf_teid) {
   magma::SetSMSessionContextAccess response;
   const auto& config = session_state.get_config();
-
+  bool flag_set      = false;

```

Comment:
can this var be more specific
Also is good to declare the variables close where they are going to be used. This way they are not in your mind when trying to understand the code

Reply:
Yes, changes done.
This flag is used for only send to AMF for  default Qos parameters [if no policy activated corresponding IMSI and APN] or getting from policydb.